src/Draco.Compiler/Api/Syntax/StringExpressionSyntax.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Linq;
+
+namespace Draco.Compiler.Api.Syntax;
+public partial class StringExpressionSyntax
+{
+    public int Padding => this.CloseQuotes.LeadingTrivia.Aggregate(0, (value, right) =>
```

Comment:
Keep in mind that padding generally doesn't make sense for multiline strings, we use a padding string for a reason. We look for exact sequence matches. If it's spaces only, then spaces only, if it's space-tab-tab-space, we look for space-tab-tab-space.